Squirrely-Ducks/bangazon-cli-sprint2
# Ed faker data

## DESCRIPTION
Generate Faker Data and insert to database

## STEPS TO TEST

1. Generating data should not be necessary since all JSON files have been pushed. However, install faker by running the command that follows. 
```javascript
$ npm install
```
- To generate data, execute the following:
```javascript
$ node data/executeData.js
```

2. To build database, run the following:
```javascript
node db/database.js
```

3. Test functionality
	* Open bangazon.sqlite in DB Browser and check for data
	

## REFERENCE
[Ticket - Generate Faker data and populate the database](https://github.com/Squirrely-Ducks/bangazon-cli-sprint2/projects/1#card-8702128)
